Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -631,6 +632,21 @@ vector<EventsSearchResult> EventsRefactorer::SearchInEvents(
       }
     }
 
+    if (inComments) {
+      vector<gd::String> stringsVectors = events[i].GetAllSearchableStrings();
+      
+      for (std::size_t j = 0; j < stringsVectors.size(); ++j) {
+        if (!eventAddedInResults &&
+            SearchStringInComments(

```

Comment:
Pass just the event, not a string:

`project, layout, events[i], search, matchCase`